pkg/proxy/nftables/proxier.go

```diff
@@ -55,25 +56,25 @@ import (
 
 const (
 	// the services chain
-	kubeServicesChain utiliptables.Chain = "KUBE-SERVICES"
```

Comment:
IMO the big reason the utiliptables types are good is that without them, we would have APIs like

```
        EnsureRule(string, string, string, ...string)
```

and you'd never be able to remember which argument was which. But having argument-specific types fixes that:

```
        EnsureRule(RulePosition, Table, Chain, ...string)
```

But the knftables APIs are mostly based on structs:

```
        tx.Add(&knftables.Rule{
                Chain:   myChainName,
                Rule:    "my rule ...",
                Comment: ptr.To("my comment"),
        })
````

so they're much more self-documenting / non-confusable even without argument-specific types.

Sure, we _could_ make the `Chain` field there be a `knftables.ChainName` type, but I'm not sure it would help anyone. It would stop people from putting a table name where a chain name belonged, but it wouldn't stop people from putting the _wrong_ chain name there, which seems like a more likely bug...